docs/builtins.rst

```diff
@@ -0,0 +1,217 @@
+================
+Builtins in L20n
+================
+
+L20n ships with a number of built-in functions that can be used from within
+localization messages.
```

Comment:
I'm missing a sentence about the purpose of `Functions`.  Something about formatting data according to the current language's rules would be enough.